docs/conf.py

```diff
@@ -26,6 +28,19 @@
 import sphinx_rtd_theme
 from packaging.version import parse
 
+# setting the metaclass will cause sphinx
+# to document the signature of `__call__`
+# rather than `__init__` that is unhelpful
+# for instruments. When building the docs
+# we patch it back to ABCMeta
+import qcodes.instrument.instrument_meta
+
+qcodes.instrument.instrument_meta.InstrumentMeta = ABCMeta
+reload(qcodes.instrument.instrument)
+reload(qcodes.instrument.ip)
+reload(qcodes.instrument.visa)
```

Comment:
would a little comment saying "we need to reload all the modules that import InstrumentMeta" here be useful for anyone reading/developing/improving?